crates/node/src/shell/governance.rs

```diff
@@ -429,12 +429,25 @@ where
                 shell.state.commit_tx();
                 Ok(true)
             }
+            Some(Err(e)) => {
+                tracing::warn!(
+                    "Error executing governance proposal {}",
+                    e.to_string()
+                );
+                shell.state.drop_tx();
+                Ok(false)
+            }
             _ => {
+                tracing::warn!("not sure what happen");
```

Comment:
Let's log this as an error - it's when the tx result is missing for the commitment hash